rpc.c

```diff
@@ -577,6 +578,7 @@ rpc_client_connect (rpc_instance rpc, const char *url)
         if (client->sock != NULL && !client->sock->dead)
         {
             /* This client will do */
+            client->timeout = timeout;
```

Comment:
I made it so rpc_client_connect becomes a wrapper for this function, and passes it RPC_TIMEOUT_US as the timeout value, so I think that every time rpc_client_connect is called, the timeout will be reset to the default?